### ISSUE TITLE
textpad - selection done using shift + arrow keys is not read

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4535]
**Reported by manish on 2014-10-07 01:26**
In the text pad app, NVDA doesn't read anything when text is selected or unselected using shift + arrow keys.
**Blocking #4790**

**Comment by bhavyashah** [comment]
Something similar happens in Excel as noted in another ticket also. Selection via Shift+arrow keys within Excel cells is not reported either. Are these related in any way?

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by Adriani90** [comment]
@manish are you still able to reproduce the issue?

**Comment by Adriani90** [comment]
I am testing in NVDA 2019.1.1 and Textpad 8.1. The issue is still reproducible. Text selections are not reported at all.

cc: @leonardder this seems to be yet another display model case.

**Comment by Adriani90** [comment]
With NVDA alpha-32879,41418d7a (2024.4.0.32879) and Textpad 9.5, the issue in https://github.com/nvaccess/nvda/issues/4535#issuecomment-155328935 is still reproducible.

**Comment by Adriani90** [comment]
It seems the developers of Textpad have overwritten the standard selection keyboard commands. This needs an external fix, the standard windows selection keyboard commands should be adopted.
